package.json

```diff
@@ -86,11 +88,12 @@
     "to-single-quotes": "^1.0.2",
     "vow": "~0.4.8",
     "vow-fs": "~0.3.4",
-    "xmlbuilder": "^2.6.1",
-    "htmlparser2": "3.8.3"
+    "xmlbuilder": "^2.6.1"
   },
   "devDependencies": {
     "browserify": "^11.0.0",
+    "chai": "^3.3.0",
+    "chai-string": "^1.1.3",
```

Comment:
I used `startsWith`. Couldn't use `match` with automatic test rewriting (i.e. converting to regexp is tricky).